lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -108,13 +108,15 @@ abstract class BaseAdapter {
 
   final DomRenderer domRenderer;
   PointerDataCallback _callback;
-  Map<int, bool> _isDownMap = <int, bool>{};
-  bool _isButtonDown(int button) {
-    return _isDownMap[button] == true;
+  Map<int, Map<int, bool>> _isDownMap = <int, Map<int,bool>>{};
+  bool _isButtonDown(int device, int button) {
+    _isDownMap.putIfAbsent(device, () => <int, bool>{});
```

Comment:
One concern I have here is the map will keep growing indefinitely. Especially in browsers that support `PointerEvent`s and provide a different `pointerId` for each new touch.

Reply:
How about keeping a set of (device_id, button) pairs: being in the set means that the device_id has that button down. When the button comes up then we take it out of the set

Reply:
Yep, I think that would work!

Reply:
Done.